plugin-example/README.md

```diff
@@ -0,0 +1,7 @@
+# Sapio Plugin Examples
+
+To run plugin examples, you must first have zig and wasm-pack installed.
+
+Then run: `CC="zig cc" wasm-pack build`
```

Comment:
not sure that Zig is required, but doesn't seem like the worst idea to use it as CC. Can you say more on why?

Reply:
It was something I was told... I can try validating that assumption